### DISCUSSION TITLE
Per-player time

### DISCUSSION THREAD

**Discussion by EpicPlayerA10** [Minestom/Minestom#897] [Help] [answered]
Is there a way to set per-player time? I tried to send a `TimeUpdatePacket` to player but server keeps sending real instance time.

**Comment by mworzala** [comment, marked as answer]
Should be fine to set the instance time rate to zero (https://javadoc.minestom.net/net/minestom/server/instance/Instance.html#setTimeRate(int)) and then send the time packet yourself.
